nixos/modules/services/networking/v2raya.nix

```diff
@@ -33,7 +44,7 @@ with lib;
 
         serviceConfig = {
           User = "root";
-          ExecStart = "${getExe pkgs.v2raya} --log-disable-timestamp";
+          ExecStart = "${getExe (pkgs.v2raya.override { v2ray = config.services.v2raya.cliPackage; })} --log-disable-timestamp";
```

Comment:
```suggestion
          ExecStart = "${getExe cfg.package; })} --log-disable-timestamp";
```

Reply:
this doesn't do what you think it does, since the package is v2raya, not xray so this change will break the execstart